Base/QTCore/qSlicerCoreApplication.cxx

```diff
@@ -278,6 +279,11 @@ void qSlicerCoreApplicationPrivate::init()
 
   QCoreApplication::setOrganizationDomain(Slicer_ORGANIZATION_DOMAIN);
   QCoreApplication::setOrganizationName(Slicer_ORGANIZATION_NAME);
+  QIcon icon;
```

Comment:
Since `QIcon` is provided by `Qt GUI`[^1], what do you think of moving the code into `qSlicerApplicationPrivate::init()` ?

[^1]: https://doc.qt.io/qt-5/qicon.html

Reply:
Thanks for the review. Yes we can move the code there. I've pushed the corresponding update.